lib/css/components/_stacks-activity-indicator.less

```diff
@@ -13,9 +13,17 @@
 
 .s-activity-indicator {
     display: inline-block;
-    width: @su12;
+    text-align: center;
+    min-width: @su12;
     height: @su12;
+    padding-left: @su4;
+    padding-right: @su4;
+    line-height: 1;
```

Comment:
At viewport widths < `lg`, line height gets kinda jacked up.

![20210907-112332](https://user-images.githubusercontent.com/647177/132370652-d8233af8-5cec-47c0-99ab-94d0d5ddbc8a.png)

At other widths, the vertical negative space is minimal to the point of impacting legibility IMO.

![20210907-112547](https://user-images.githubusercontent.com/647177/132371046-0d519c6d-4acc-4148-8897-e80fd5937cdb.png)

I'd changing the `height` to a `min-height` and setting an increased `line-height` for this result:

![20210907-112707](https://user-images.githubusercontent.com/647177/132371227-2f0ca2a8-90fe-4d7b-be15-b2267f93e3d2.png)

```suggestion
    min-width: @su12;
    min-height: @su12;
    padding-left: @su4;
    padding-right: @su4;
    line-height: @lh-md;
```

Reply:
BTW, this will increase the height of the activity indicator from 12px to about 14px when it has text inside. I imagine that should be fine but I figured I should call that out.

Reply:
I can't repro this, is it a `1x` screen issue? 🤔

Reply:
Ah, yep. Shoved my browser onto my high DPR screen and looks better there.

### At 1x (despite what Firefox says)

<img width="375" src="https://user-images.githubusercontent.com/647177/132377948-679fb04f-6f6b-4cf9-82ee-ffe8935745e2.png" />

### At 2x

<img width="375" alt="20210907-120957" src="https://user-images.githubusercontent.com/647177/132378062-d4472695-96d4-4b1e-8c20-45af3fde8682.png">

Reply:
Upon further investigation, this may be a Gecko-only `1x` issue. I checked Chrome/Safari and no weird line-height found there.

Reply:
**tl;dr**:  I'd suggest we use `line-height: 1.1;` and move on with our lives (after checking different OSes/browsers/fonts in our stack for issues, of course).

---

This issue happens specifically with `line-height: 1;`. It seems that just about anything between `1` and `@lh-sm` could work but introducing a new value for this edge case feels just a little icky. I really wanted `@lh-sm` to work, but it shifts the text down a pixel.

## Examples

### `line-height: 1;`

![20210923-042544](https://user-images.githubusercontent.com/647177/134580207-6af82ca7-8752-4cd5-8d65-ce5b48c62592.gif)

### `line-height: 1.1;`

![20210923-043326](https://user-images.githubusercontent.com/647177/134580245-6e2bc26f-b0f0-4a6e-942d-2b3ecf6c2999.gif)

Reply:
How about now, @dancormier?

Reply:
👍 LGTM